### ISSUE TITLE
Feature: Add --braindmp-path option

### ISSUE THREAD

**Issue by f0cker** [hashcat/hashcat#2550]
It would be useful to be able to select the path to save brain dump files. I'll submit a PR for this myself, but just want to raise the issue first for comments before I do.

**Comment by philsmd** [comment]
Yeah, I wouldn't be against it. we added a few other parameters that have to do with the brain-server recently like `--brain-server-timer` that allows configuring the time between "backups"/writes etc.

I'm just not completely convinced about the naming, but also have no perfect suggestion for you.
I think best would be if we use "--brain-server-*" within the name... maybe something like `--brain-server-dump-dir` ?

I'm also not completely sure if we should use `-dir`, `-file` or `-path`... if I look at the current `--help` output we use `*-file` and `*-path` only for single files, not for a directory ....

That brings us to another question, do the user need to store the 2 different files separately and in very different locations / files / dirs ?

I think to not overcomplicate it, a `*-dir` parameter would be enough for the start !? This should of course work for both loading old backups, but also for writing to files within that folder (the actual file name will depend on session etc).

Thx
